skimage/util/montage.py

```diff
@@ -108,3 +121,68 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
     arr_out = arr_out.reshape(alpha_y * height, alpha_x * width)
 
     return arr_out
+
+
+def montage_rgb(arr_in, fill='mean', grid_shape=None, padding_width=0):
```

Comment:
@sciunto I agree that there is no clear use for nD case here, but it should be relatively easy to generalize the function.
As for `multichannel`, I see a nice use for this function in multispectral satellite imaging.